e2e/cypress/integration/dashboard-transloadit.spec.ts

```diff
@@ -65,14 +65,14 @@ describe('Dashboard with Transloadit', () => {
       cy.get('.uppy-StatusBar-actionBtn--upload').click()
 
       cy.wait(['@createAssemblies', '@tusCreate']).then(() => {
-        const plugin = getPlugin(uppy)
+        const { assembly } = getPlugin(uppy)
 
-        expect(plugin.assembly.closed).to.be.false
+        expect(assembly.closed).to.be.false
 
         uppy.cancelAll()
 
         cy.wait(['@delete', '@tusDelete']).then(() => {
-          expect(plugin.assembly.closed).to.be.true
+          expect(assembly.closed).to.be.true
```

Comment:
Those changes are purely aesthetic, right?

Reply:
nope, because `plugin.assembly` is now set to `undefined` after an upload has finished, we need to get the assembly out first, or else the test was failing due to assembly being undefined on plugin